quickstarts/microsoft.compute/vm-cyclecloud/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "ccadminRawPassword": {
+      "value": "TODO"
```

Comment:
See https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#parameters-file-placeholders

Reply:
Added secure decorator and used GEN-PASSWORD for value.